examples/jsm/loaders/LDrawLoader.js

```diff
@@ -17,6 +18,8 @@ import {
 	Ray
 } from '../../../build/three.module.js';
 
+import { mergeBufferGeometries } from '../utils/BufferGeometryUtils.js';
```

Comment:
Small note that this is maybe a bit related to #23167 now that it's importing a utils function and is a breaking change for anyone using the traditional global import approach.

Reply:
Yes, definitely it is related. Is there anymore that can be done in this PR? Could the utility function ```mergeObject``` be placed in another class to not disturb LDrawLoader?

Reply:
I think it's okay as-is. Was just pointing it out. Maybe it should be added to the migration guide?

Reply:
I'd feel more comfortable by creating a new file ```jsm/utils/LDrawUtils.js``` and moving the function there. I'd also create a documentation file for LDrawLoader with explanations on the loader, the ```LDrawUtils``` and the ```packLDrawModel``` utility. Let me know if you agree @gkjohnson @mrdoob

Reply:
I'll defer to @mrdoob!

Reply:
`LDrawUtils` sounds good 👍